cogs/basic.py

```diff
@@ -35,15 +35,18 @@ def __init__(self, bot):
 
     @commands.command(name='xkcd')
     @commands.cooldown(5, 10, commands.BucketType.user)
-    async def get_xkcd(self, ctx):
+    async def get_xkcd(self, ctx, id=None):
```

Comment:
```suggestion
    async def get_xkcd(self, ctx, id: int=None):
```
Since only integer values are valid, let's use [type annotations so discord.py knows to only accept ints.](https://discordpy.readthedocs.io/en/rewrite/ext/commands/commands.html#basic-converters)